client/coral-admin/src/containers/Configure.js

```diff
@@ -40,12 +42,28 @@ class Configure extends React.Component {
     </List>
   }
 
+  copyToClipBoard (event) {
+    const copyTextarea = document.querySelector('.' + styles.embedInput)
+    copyTextarea.select()
+
+    try {
+      document.execCommand('copy')
+    } catch (err) {
+      console.error('Unable to copy')
+    }
+  }
+
   getEmbed () {
+    const embedText =
+    `<div id='coralStreamEmbed'></div><script type='text/javascript' src='https://pym.nprapps.org/pym.v1.min.js'></script><script>var pymParent = new pym.Parent('coralStreamEmbed', '${window.location.protocol}//${window.location.host}/client/coral-embed-stream/', {title: 'comments'});</script>`
```

Comment:
I like this pattern a lot. Embed links per install with no configuration needed!